packages/components/pie-tag/package.json

```diff
@@ -13,7 +13,7 @@
     "**/*.d.ts"
   ],
   "pieMetadata": {
-    "componentStatus": "beta"
+    "componentStatus": "stable"
```

Comment:
we are introducing a breaking change for the tag component on this PR
https://github.com/justeattakeaway/pie/pull/2023, should we update it there instead so we don't have to release two majors?

Reply:
Good point. I think we can make the "major" change in both places but we should make sure to release them as part of the same changeset.

Reply:
Although actually I would argue that `pie-tag` is not stable if we're things like this...

Reply:
great spot @raoufswe - reverted tag